tools/ldb_test.py

```diff
@@ -340,6 +366,24 @@ def testDumpLoad(self):
         self.assertFalse(self.dumpDb(
             "--db=%s --create_if_missing" % origDbPath, dumpFilePath))
 
+        # Dump and load with BlobDB enabled
+        blobParams = " ".join(["--enable_blob_files", "--min_blob_size=1",
+                                "--blob_file_size=2097152"])
+        dumpFilePath = os.path.join(self.TMP_DIR, "dump9")
+        loadedDbPath = os.path.join(self.TMP_DIR, "loaded_from_dump9")
+        self.assertTrue(self.dumpDb(
+            "--db=%s %s" % (origDbPath, blobParams), dumpFilePath))
```

Comment:
Minor but you don't have to pass `blobParams` to `dump` since it has no effect in this case.